ui-components/src/headers/StepHeader.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react"
+
+import { StepHeader } from "./StepHeader"
+
+export default {
+  title: "Headers/Step Header 🚩",
+}
+
+export const defaultTitle = () => <StepHeader currentStep={1} totalSteps={4} />
+
+export const customTitle = () => <StepHeader currentStep={1} totalSteps={4} stepTitle={"Phases"} />
+
+export const finalStep = () => <StepHeader currentStep={3} totalSteps={3} />
+
+export const doubleDigits = () => <StepHeader currentStep={10} totalSteps={40} />
```

Comment:
Are 3 digits in scope? If so, looks like those get cut off 
<img width="189" alt="image" src="https://user-images.githubusercontent.com/42942267/183456665-6c71706e-9aa8-4103-904c-b674a20e89c1.png">.
A potential fix is to switch the circle-number css from "width" to "min-width"

Reply:
Appreciate the feedback! While I couldn't envision it in the current planned usage, it seemed worth it for potential future uses. Let me know what you think about the recent changes.